src/Game/MeshBuilder.h

```diff
@@ -2,7 +2,40 @@
 
 #include "../Renderer/Mesh.h"
 #include "WorldTextures.h"
+#include "CYObjects.h"
 
 struct Wall;
 
-Mesh createMesh(const Wall& wall, const WorldTextures& wTex);
\ No newline at end of file
+Mesh createMesh(const Wall& wall, const WorldTextures& wTex);
+
+namespace {
```

Comment:
why was this moved to the header file?

Reply:
I was having linker issues at the time. I put it back into the .cpp and used a namespace